solutions/swb-app/src/dataSets/DataSetFileUploadRequestParser.ts

```diff
@@ -2,12 +2,12 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
-import { z } from 'zod';
+import { resourceTypeToKey, z } from '@aws/workbench-core-base';
 
 // eslint-disable-next-line @rushstack/typedef-var
 export const DataSetFileUploadRequestParser = z.object({
-  dataSetId: z.string(),
-  filenames: z.union([z.string(), z.array(z.string())])
+  dataSetId: z.string().swbId(resourceTypeToKey.dataset).required(),
```

Comment:
please add `datasetId()` interface to zodstring.. we are not using `swbId()` anymore